docs/api/10_webhooks.mdx

```diff
@@ -207,22 +207,92 @@ It will allow your application to perform the billing.
         "amount_cents": 100,
         "amount_currency": "EUR",
         "vat_amount_cents": 20,
-        "vat_amount_currency": "EUR"
+        "vat_amount_currency": "EUR",
         "units": "0.32"
       }
+    ],
+    "credits": [
+      {
+        "item": {
+          "type": "coupon",
+          "code": "coupon_code",
+          "name": "Coupon"
+        },
+        "amount_cents": 100,
+        "amount_currency": "EUR"
+      }
     ]
-  },
-  "credits": [
-    {
-      "item": {
-        "type": "coupon",
-        "code": "coupon_code",
-        "name": "Coupon",
-      },
-      "amount_cents": 100,
-      "amount_currency": "EUR"
-    }
-  ]
+  }
+}
+```
+
+### Add-on applied to customer
+
+Sent when an add-on has been applied to customer.
+
+It will allow your application to perform the billing.
```

Comment:
Would be more accurate to say: "You can use this information to generate a PDF invoice or collect the payment."